o365-auth/src/main/java/com/microsoft/office365/auth/MSALAuthenticationCallback.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.office365.auth;
+
+import com.microsoft.identity.client.AuthenticationResult;
+import com.microsoft.identity.client.exception.MsalException;
+
+public interface MSALAuthenticationCallback {
+    void onSuccess(AuthenticationResult authenticationResult);
+    void onError(MsalException exception);
+    void onError(Exception exception);
```

Comment:
Lets remove the second "onError" declaration as it is unused and may possibly lead to confusion.